src/app/vitessce-grid-utils.js

```diff
@@ -104,7 +104,14 @@ export function createLoaders(datasets, configDescription) {
       // Fall back to JsonLoader if a loader is not found for the file type.
       const matchingLoaderClass = fileTypeToLoader[file.fileType] || JsonLoader;
       // eslint-disable-next-line new-cap
-      datasetLoaders.loaders[file.type] = new matchingLoaderClass(file);
+      const loader = new matchingLoaderClass(file);
+      if (Array.isArray(file.type)) {
+        file.type.forEach((type) => {
+          datasetLoaders.loaders[type] = loader;
+        });
+      } else {
+        datasetLoaders.loaders[file.type] = loader;
+      }
```

Comment:
Maybe doing this iteration per `url` would make more sense, caching per URL?

Reply:
One issue here I noticed is that not all loaders even have `url` keys like `raster.json` where the loader is a wrapper around Viv loaders, which are what you would really want to catch.  So I think this is out of scope here for the general case?